worker/games.py

```diff
@@ -493,11 +493,11 @@ def parse_options(s):
   if not os.path.exists(testing_dir):
     os.makedirs(testing_dir)
 
-  # clean up old engines (keeping the 25 most recent)
+  # clean up old engines (keeping the 50 most recent)
   engines = glob.glob(os.path.join(testing_dir, 'stockfish_*' + EXE_SUFFIX))
-  if len(engines) > 25:
+  if len(engines) > 50:
     engines.sort(key=os.path.getmtime)
-    for old_engine in engines[:len(engines)-25]:
+    for old_engine in engines[:len(engines)-50]:
```

Comment:
@tomtor this could be simplified in `for old_engine in engines[:-50]:`

Reply:
Thanks!